perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +80,107 @@
                   'num_machines': 1}
 
 
+JOB_FILE_TEMPLATE = """
```

Comment:
@stfeng 

I agreed that it was a new benchmark because I thought the results from the new code wouldn't be comparable to results from the old code. I just checked the job files, and actually the old and new use the same options (I think I did this on purpose), so they are more comparable than I thought.

The new jobfile generation code will test directly against the device if you use the --against_device argument.